src/DefaultBuilder/test/Microsoft.AspNetCore.Tests/WebApplicationTests.cs

```diff
@@ -2489,7 +2489,7 @@ public async Task UsingCreateSlimBuilderWorksIfRegexConstraintAddedViaAddRouting
 
     private class UberHandler : AuthenticationHandler<AuthenticationSchemeOptions>
     {
-        public UberHandler(IOptionsMonitor<AuthenticationSchemeOptions> options, ILoggerFactory logger, UrlEncoder encoder, ISystemClock clock) : base(options, logger, encoder, clock) { }
+        public UberHandler(IOptionsMonitor<AuthenticationSchemeOptions> options, ILoggerFactory logger, UrlEncoder encoder, TimeProvider time) : base(options, logger, encoder, time) { }
```

Comment:
`time` instead of `timeProvider` confuses me a lot. But maybe it's because I have written it too many times myself and I am biased.

If we have a type named `FileProvider`, would we name the fields `file` or `fileProvider` 😉 (NB: I checked)